imports/wasi_snapshot_preview1/example/testdata/tinygo/cat.go

```diff
@@ -10,12 +12,27 @@ import (
 func main() {
 	// Start at arg[1] because args[0] is the program name.
 	for i := 1; i < len(os.Args); i++ {
-		bytes, err := os.ReadFile(os.Args[i])
+		file, err := os.Open(os.Args[i])
+		if err != nil {
+			os.Exit(1)
+		}
+
+		defer file.Close()
+
+		stat, err := file.Stat()
+		if err != nil {
+			os.Exit(1)
+		}
+
+		os.Stdout.Write([]byte(fmt.Sprintf("Size: %d\n", stat.Size())))
```

Comment:
cool I trust this works so you can revert it now!

Reply:
Don't you want to keep this in here to make sure that stat keeps working? Or is it actually supposed to be an example and not some sort of test?

Reply:
the example is not a full scale test. Main thing since things aren't documented is to make sure something works with it. After that's the case normal unit tests should be enough.